tutorials/simulation_exporting.ipynb

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.        We want to override the model methods with those
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>